dapps/marketplace/test/multiUnit.js

```diff
@@ -171,6 +173,70 @@ export function multiUnitTests({
       )
     })
 
+    it('should make an offer on a new listing', async function() {
+      await purchaseMultiUnitListing({
+        page,
+        buyer,
+        withShipping,
+        title,
+        withToken: 'ETH'
+      })
+    })
+
+    it('should allow a new offer to be withdrawn', async function() {
+      await withdrawOffer({ page, buyer })
+    })
+
+    /**
+     * TODO: Is this a bug?  going home first I guess kind of requires a
+     * refetch?  If we don't, the item will show "Sold Out"
+     */
+    it('should navigate back home', async function() {
+      await page.evaluate(l => {
```

Comment:
You can prevent a full refresh by using the hash in the url... ie `/#/` instead of just `/`

Reply:
Made the change, but it doesn't appear to cause a refetch which is what I was going for here.  Listing will still show as "Sold Out" in this case.